app/controllers/profiles_controller.rb

```diff
@@ -6,7 +6,7 @@ def show
   end
 
   def new
-    @profile = Profile.new(user_id: current_user.id)
```

Comment:
If you touch code outside of your story submit it as seperate merge request, because i want you to focus on your story